File: src/Libro.java
class Libro {
    private int codigoDelLibro;
    private String titulo;
    private String autor;
    private String apellidoAutor;
    private boolean disponibilidad;

    public Libro(int codigoDelLibro, String titulo, String autor, String apellidoAutor, boolean disponibilidad) {
        this.codigoDelLibro = codigoDelLibro;
        this.titulo = titulo;
        this.autor = autor;
        this.apellidoAutor = apellidoAutor;
        this.disponibilidad = disponibilidad;
    }

    public int getCodigoDelLibro() {
        return codigoDelLibro;
    }

    public String getTitulo() {
        return titulo;
    }

    public String getAutor() {
        return autor;
    }

    public String getApellidoAutor() {
        return apellidoAutor;
    }

    public boolean getDisponibilidad() {
        return disponibilidad;
    }

    public void setDisponibilidad(boolean disponibilidad) {
        this.disponibilidad = disponibilidad;
    }
}

File: Libro.py
class Libro:
    def __init__(self, codigo_del_libro, titulo, autor, apellido_autor, disponibilidad):
        self.codigo_del_libro = codigo_del_libro
        self.titulo = titulo
        self.autor = autor
        self.apellido_autor = apellido_autor
        self.disponibilidad = disponibilidad

    def set_disponibilidad(self, disponibilidad):
        self.disponibilidad = disponibilidad